lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -73,20 +73,20 @@ static const unsigned char kernels_fwd_quantized_conv_cl_src[] = {
 static const size_t kernels_fwd_quantized_conv_cl_src_size =
     sizeof(kernels_fwd_quantized_conv_cl_src);
 
-namespace {
 llvm::cl::OptionCategory OpenCLBackendCat("Glow OpenCL Backend Options");
 
-static llvm::cl::opt<unsigned>
-    platformId("platform", llvm::cl::desc("OpenCL platform to be used"),
-               llvm::cl::init(0), llvm::cl::cat(OpenCLBackendCat));
-static llvm::cl::opt<unsigned>
-    deviceId("device", llvm::cl::desc("OpenCL device to be used"),
-             llvm::cl::init(0), llvm::cl::cat(OpenCLBackendCat));
-static llvm::cl::opt<bool> doProfile("opencl-profile",
-                                     llvm::cl::desc("Profile OpenCL kernels"),
-                                     llvm::cl::init(false),
-                                     llvm::cl::cat(OpenCLBackendCat));
-} // namespace
+llvm::cl::opt<unsigned> platformId("platform",
```

Comment:
Since this options are global now, may be the names of the variables should be more OpenCL specific? `platformId` and `deviceId` seems too generic for a global variable.

Reply:
changed to clPlatformId, clDeviceId, and clDoProfile.